pkg/fieldpath/paved.go

```diff
@@ -120,6 +124,80 @@ func (p *Paved) getValue(s Segments) (interface{}, error) {
 	return nil, nil
 }
 
+// ExpandWildcards expands wildcards for a given field path. It returns an
+// array of field paths with expanded values. Please note that expanded paths
+// depend on the input data which is paved.object.
+//
+// Example:
+//
+// For a Paved object with the following data: []byte(`{"spec":{"containers":[{"name":"cool", "image": "latest", "args": ["start", "now", "debug"]}]}}`),
+// ExpandWildcards("spec.containers[*].args[*]") returns:
+// []string{"spec.containers[0].args[0]", "spec.containers[0].args[1]", "spec.containers[0].args[2]"},
+func (p *Paved) ExpandWildcards(path string) ([]string, error) {
+	segments, err := Parse(path)
+	if err != nil {
+		return nil, errors.Wrapf(err, "cannot parse path %q", path)
+	}
+	segmentsArray, err := expandWildcards(p.object, segments)
```

Comment:
Would it make sense to create `GetValues(path string) ([]interface{}, error)` (and other variants of it working with array) and call `expandWildcards` and `GetValue` functions?

Reply:
I am a bit hesitant to implement each variant at this moment, given it is already possible to first expand values and then iterate over returned fieldpaths and use existing Get* methods. I would prefer to consider it later if that is needed.